mlflow/sklearn/utils.py

```diff
@@ -302,6 +565,16 @@ def _is_supported_version():
     return LooseVersion(sklearn.__version__) >= LooseVersion(_MIN_SKLEARN_VERSION)
 
 
+# Util function to check whether a metric is able to be computed in given sklearn version
+def _is_metric_supported(metric_name):
+    import sklearn
+
+    # This dict can be extended to store special metrics' specific supported versions
+    _metric_supported_version = {"training_roc_auc_score": "0.22.2"}
```

Comment:
```suggestion
    _metric_supported_version = {"roc_auc_score": "0.22.2"}
```
Should we exclude `training_` becase `"training_roc_auc_score": "0.22.2"` souds like scikit-learn 0.22.2 has a metric function named `traning_roc_auc_score`?

Reply:
Makes sense!